contracts/NFTMarket.sol

```diff
@@ -281,7 +281,7 @@ contract NFTMarket is
         EnumerableSet.UintSet storage set = listedTokenIDs[address(_tokenAddress)];
         uint256 matchingCharactersAmount = getNumberOfCharacterListings(_tokenAddress, _trait, _minLevel, _maxLevel);
         uint256 pageEnd = _limit * (_pageNumber + 1);
-        uint256 tokensSize = matchingCharactersAmount >= pageEnd ? _limit : matchingCharactersAmount - (_limit * _pageNumber);
+        uint256 tokensSize = matchingCharactersAmount >= pageEnd ? _limit : uint256(SafeMath.sub(matchingCharactersAmount, _limit * _pageNumber));
```

Comment:
Use "member-like" variant of `SafeMath` function calls.

```suggestion
        uint256 tokensSize = matchingCharactersAmount >= pageEnd ? _limit : matchingCharactersAmount.sub(_limit * _pageNumber);
```

Reply:
Oh I see I could just use your commit suggestion, just discoverd this, thanks.